src/test/java/vehicle/GetVehicleTechnicalRecords.java

```diff
@@ -469,4 +469,34 @@ public void testVehicleTechnicalRecordsGetAllVrms() {
         vehicleTechnicalRecordsSteps.valueForFieldInPathShouldBe("vrms[1].vrm", "CT96DRG");
         vehicleTechnicalRecordsSteps.valueForFieldInPathShouldBe("vrms[1].isPrimary", false);
     }
+
```

Comment:
These new tests are already covered in 10217 related to tech record changes, so we should remove them to not have duplicates.